### PR TITLE
ReleaseNotes: Updated changelog and release notes for 8.2.0-beta1

### PR DESCRIPTION
**Pull request by grafanabot**
This exciting! So much has changed!

### CODE REVIEW

**Review by torkelo — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -39,9 +43,12 @@
 - **Postgres/MySQL/MSSQL:** Add setting to limit the maximum number of rows processed. [#38986](https://github.com/grafana/grafana/pull/38986), [@marefr](https://github.com/marefr)
 - **Prometheus:** Add browser access mode deprecation warning. [#37578](https://github.com/grafana/grafana/pull/37578), [@ivanahuckova](https://github.com/ivanahuckova)
 - **Prometheus:** Add interpolation for built-in-time variables to backend. [#39051](https://github.com/grafana/grafana/pull/39051), [@ivanahuckova](https://github.com/ivanahuckova)
+- **Recorded Queries:** Finish rest API endpoints. (Enterprise)
+- **Reporting:** enable creating reports from dashboard. (Enterprise)
```
Comment (osg-grafana) [inline CHANGELOG.md:47]: @torkelo can we (easily enough) pull in URLs for issues such as these?

File: CHANGELOG.md [hunk 2]
```diff
@@ -57,17 +64,26 @@
 
 ### Breaking changes
 
+
 The `monaco-editor` dependency in `grafana-ui` has been updated to a newer version (`0.27.0`), which is not completely backward compatible with the old version (`0.21.2`). The backward incompatible changes are fairly small, but they do exist, so if your code accesses the raw monaco-objects through the `grafana-ui` package, please check the [monaco-editor changelog](https://github.com/microsoft/monaco-editor/blob/main/CHANGELOG.md) and apply any necessary changes. Issue [#39027](https://github.com/grafana/grafana/issues/39027)
 
-The mandatory `css` prop in `grafana/ui` components has been removed.
+
+The mandatory `css`  prop in `grafana/ui` components has been removed.
```
Comment (osg-grafana) [inline CHANGELOG.md:71]:
```suggestion
The mandatory `css` property in `grafana/ui` components has been removed. 
```
Write a complete word or term.

**Review by osg-grafana — CHANGES_REQUESTED** [review]
* Use the imperative (command) form of the verb.
* Write a complete word or term.

**Review by osg-grafana — APPROVED** [review]
Approving, given feedback is incorporated and RNs are rerun to get upstream changes.

File: CHANGELOG.md (hunk 1 above)
Comment (torkelo) [reply CHANGELOG.md:47]: no, Enterprise repo is private, and we do not want to link to private issues here

File: CHANGELOG.md (hunk 2 above)
Comment (torkelo) [reply CHANGELOG.md:71]: updated this text in the source PR https://github.com/grafana/grafana/pull/38078

**Review by torkelo — APPROVED** [review]

**Pull request merged**